app/code/Academy/SmsSubscription/view/adminhtml/templates/message/form.phtml

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Using .phtml forms inside the admin panel works but they need a lot of styling to look nice and usually, it's difficult to make them look similar to each other and to keep the Magento 2 feel. UI component forms are the preferred option because they get the Magento 2 styling, look nice out of the box, and are theme-independent.